### PR TITLE
Add postprocessing to show TOF diagnostic output per crate

### PR DESCRIPTION
**Pull request by njacazio**
- Add postprocessing to show TOF diagnostic output per crate
- include minor fixes for logging messages

### CODE REVIEW

File: Modules/TOF/src/PostProcessDiagnosticPerCrate.cxx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        }
+      }
+    }
+  }
+  ILOG(Info) << "DONE UPDATING !";
+}
+
+void PostProcessDiagnosticPerCrate::finalize(Trigger, framework::ServiceRegistry&)
+{
+  ILOG(Info) << "FINALIZING !";
```
Comment (knopers8) [inline Modules/TOF/src/PostProcessDiagnosticPerCrate.cxx:88]:
```suggestion
  ILOG(Info) << "FINALIZING !" << ENDM;
```

File: Modules/TOF/src/PostProcessDiagnosticPerCrate.cxx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    TCanvas* c = new TCanvas(i->GetName(), i->GetName());
+    i->Draw();
+    auto mo = std::make_shared<o2::quality_control::core::MonitorObject>(c, "PostProcessDiagnosticPerCrate", "TOF");
+    mo->setIsOwner(false);
+    mDatabase->storeMO(mo);
+
+    // It should delete everything inside. Confirmed by trying to delete histo after and getting a segfault.
+    delete c;
+  }
+  ILOG(Info) << "DONE FINALIZING !";
```
Comment (knopers8) [inline Modules/TOF/src/PostProcessDiagnosticPerCrate.cxx:100]:
```suggestion
  ILOG(Info) << "DONE FINALIZING !" << ENDM;
```

File: Modules/TOF/src/PostProcessDiagnosticPerCrate.cxx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   PostProcessDiagnosticPerCrate_OLD.cxx
```
Comment (knopers8) [inline Modules/TOF/src/PostProcessDiagnosticPerCrate.cxx:12]:
```suggestion
/// \file   PostProcessDiagnosticPerCrate.cxx
```

**Review by knopers8 — CHANGES_REQUESTED** [review]
Hello, it is nice to see how the post-processing framework is used. I have some cosmetic change suggestions, otherwise it seems good.

You may want to consider using ObjectManager to store the objects. This should enable you to add checks to them in the near future. However, the original way to do it is still fine.
As an example, you can see the recent changes in TrendingTask: https://github.com/AliceO2Group/QualityControl/pull/497/files#diff-0e867c74aa9bf3a844f0d904aa27f46b

**Comment by njacazio** [comment]
@knopers8 thanks for your input! We will defenitively move to the ObjectManager but probably in the next iteration when we want to perform checks on the reshuffled histograms.

**Review by knopers8 — APPROVED** [review]

**Pull request merged**